client/containers/Pub/SpubHeader/SpubHeaderToolBar/SpubHeaderToolbar.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { Dispatch, SetStateAction, useMemo } from 'react';
+import { Tab, Tabs, TabId, Icon, IconName, Button } from '@blueprintjs/core';
+import Color from 'color';
+
+import { GridWrapper } from 'components';
+import { SubmissionStatus } from 'types';
+import { usePageContext, usePendingChanges } from 'utils/hooks';
+
+require('./spubHeaderToolbar.scss');
+
+const renderTabTitle = (icon: IconName, title: string) => (
+	<>
+		<Icon icon={icon} /> {title}
+	</>
+);
+
+type Props = {
+	selectedTab: TabId;
+	onSelectTab: Dispatch<SetStateAction<TabId>>;
+	status: SubmissionStatus;
+	showSubmitButton: boolean;
+	onSubmit: () => any;
+};
+
+const SpubHeaderToolbar = (props: Props) => {
+	const instructionTabTitle = renderTabTitle('align-left', 'Instructions');
+	const submissionTabTitle = renderTabTitle('manually-entered-data', 'Submission');
+	const contributorsTabTitle = renderTabTitle('people', 'Contributors');
+	const previewTabTitle = renderTabTitle('eye-open', 'Preview');
+	const maybeActiveClass = (tabId: string) =>
+		`${tabId === props.selectedTab ? 'active' : 'inactive'}`;
+
+	const { communityData } = usePageContext();
+
+	const lighterAccentColor = useMemo(
+		() => Color(communityData.accentColorLight).alpha(0.2),
+		[communityData.accentColorLight],
+	);
+
+	const darkerAccentColor = useMemo(
```

Comment:
I goofed up in the spec for this task and claimed that this was a darker version of `accentColorDark`. As you correctly inferred from the Figma mockup, this is _exactly_ `accentColorDark`...so we can drop this altogether and just use the `--community-accent-dark` CSS variable.